common/app/model/meta.scala

```diff
@@ -648,6 +648,7 @@ final case class Tags(
 
   // Specific Series
   lazy val isLabourLiverpoolSeries = tags.exists(t => t.id == "membership/series/labour-liverpool")
```

Comment:
Is this still being used? If not, it should be removed. If it is, then it should still be part of the isHeroic definition in common/app/model/content.scala